corehq/apps/users/util.py

```diff
@@ -347,6 +346,9 @@ def log_user_change(domain, couch_user, changed_by_user, changed_via=None,
     :param domain_required_for_log: set to False to allow domain less log for specific changes
     """
     from corehq.apps.users.models import UserHistory
+    from corehq.apps.users.model_log import UserModelAction
+
+    action = action or UserModelAction.UPDATE
```

Comment:
Not pleased with this, but the circular import was pretty nasty. happy to reconsider and make a better effort if need be.
Just wanted to unblock staging deploy asap.

Reply:
I don't feel strongly about this, but since it came up...does `model_log.py` really need to be its own file? Are you expecting to add much else to that file? Seems like it could be part of users.models - which is admittedly already huge, but if adding < 10 lines to it would simplify imports, it seems worth the tradeoff.

Reply:
Okay, i gave that a shot, moved model_log to models but still ran into circular import. I also tried the opposite by moving everything related to log to model_log (which I ideally should) but still ran into some.

Its possible the solution is to change the order of apps being loaded, which I am not sure if a safe change to do, to solve a circular import.

Reply:
> Are you expecting to add much else to that file? 

while trying to move code to avoid circular import, I think it makes sense to move `log_user_change` (along with `_get_changed_details`) and `log_user_role_update` to this file as file, these are in util as of now which seems an okay place as well.

Reply:
Got it, that sounds reasonable. When first commenting, I had forgotten that `model_log.py` already existed, just in a different place.